images/linux/scripts/helpers/install.sh

```diff
@@ -12,11 +12,17 @@ download_with_retries() {
     local DEST="${2:-.}"
     local NAME="${3:-${URL##*/}}"
 
+    if [ $4 ]; then
```

Comment:
maybe use something like this to stay on a safe side?
if [ $4="-compressed" ]

Reply:
Done